mocktracer/src/mock_span_context.cpp

```diff
@@ -21,12 +21,20 @@ void MockSpanContext::ForeachBaggageItem(
   }
 }
 
-void MockSpanContext::SetData(SpanContextData& data) {
+void MockSpanContext::SetData(SpanContextData& data) const {
```

Comment:
This name is a bit confusing. Sounds like you are setting the data member named `data`. Maybe `CopyData`?

Reply:
Sure. I'll rename it.